.gitignore

```diff
@@ -1 +0,0 @@
-node_modules/
```

Comment:
Feel free to do that if you want. My 2 cents is that it's still a full repo/package that deserves a .gitignore :)